dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/view_cms_maintenance.jsp

```diff
@@ -1428,19 +1429,21 @@ dd.leftdl {
                     <th><%= LanguageUtil.get(pageContext,"Import/Export-dotCMS-Content") %></th>
                     <th style="text-align:center;white-space:nowrap;" width="350"><%= LanguageUtil.get(pageContext,"Action") %></th>
                 </tr>
-                <tr>
```

Comment:
This needs to be wrapped in a Config.getBooleanProperty - off by default but should be allowed to be turned back on in environments that require it.